scripts/alphabetiseAllTranslations.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
+
+for f in $BASEDIR/src/RMP/Translate/lang/*/*; do
```

Comment:
Rather than running this on invidiual language files, I would sort the template.pot and then run updateTranslationsFromTemplate.sh to cascade to the individual language files. It's likely to be more reliable and avoids anything getting out of sync due to errors/incosistencies in the sorting script.